strip.lua

```diff
@@ -57,7 +57,7 @@ function right()
 end
 
 function checkFuel()
-    return turtle.getFuelLevel() > 500
+    return turtle.getFuelLevel() > state.branch.length + state.trunk.length
```

Comment:
```suggestion
    return turtle.getFuelLevel() > state.branch.length + state.trunk.length + 10
```
We want a fudge factor in here as some states can not be left early such as mine